From oliver@neukum.org  Fri Feb  6 15:52:36 2009
From: Oliver Neukum <oliver@neukum.org>
Date: Fri, 6 Feb 2009 15:37:14 +0100
Subject: USB: serial: export symbol of usb_serial_generic_resume
To: Greg KH <greg@kroah.com>
Message-ID: <200902061537.14747.oliver@neukum.org>
Content-Disposition: inline


This exports a symbol for usb_serial_generic_resume, so that modules can
use it.

Signed-off-by: Oliver Neukum <oneukum@suse.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/usb/serial/generic.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/generic.c
+++ b/drivers/usb/serial/generic.c
@@ -188,6 +188,7 @@ int usb_serial_generic_resume(struct usb
 
 	return c ? -EIO : 0;
 }
+EXPORT_SYMBOL_GPL(usb_serial_generic_resume);
 
 void usb_serial_generic_close(struct tty_struct *tty,
 			struct usb_serial_port *port, struct file *filp)
